MM/5.8.0/config/media_manager_5/fields/creative_cloud_connector_show_all_asset_type_filter_options.dcl

```diff
@@ -0,0 +1,10 @@
+resource configservice_bit_config_field creative_cloud_connector_show_all_asset_type_filter_options {
+    default_value = true
+    product_id = resource.configservice_product.media_manager_5.id
+    group = 'Creative Cloud Connector'
+    key = 'creativeCloudConnectorShowAllAssetTypeFilterOptions'
+    title = 'Show all asset type filter options'
+    description = 'When enabled, you can filter on all asset types. When disabled, you can only filter on asset types included in the "Allowed asset types" field.'
```

Comment:
Aren't you missing the "Allowed asset types" field?

Reply:
The field was created in another PR